README.md

```diff
@@ -1,5 +1,11 @@
 # OpenTok Node SDK
 
+> [!IMPORTANT]
+> ⚠️ This SDK is intended **only for existing TokBox/OpenTok customers**. New developers should use the [Vonage Server SDK for Node](https://github.com/vonage/vonage-node-sdk) instead.
+
+> [!WARNING]
+> This SDK is in maintenance mode and will only receive bug fixes. New Video API features will only be added to the [Vonage Server SDK for Node](https://github.com/vonage/vonage-node-sdk)
```

Comment:
This is (unfortunately) not true, whether a feature is added or not is based on the complexity offering it to OpenTok customers.